### PR TITLE
Collect minor improvements for tremor-query docs

### PR DESCRIPTION
**Pull request by anupdhml**

### CODE REVIEW

File: docs/internal/LocalMkdocsSetup.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ pip3 install -r requirements.txt
 
 This installs the `mkdocs-material` theme which we use, along with other dependencies.
 
+To enable syntax-highlighting for tremor-script/trickle code snippets, install our mkdocs-specifc tremor lexer as well:
```
Comment (mfelsche) [inline docs/internal/LocalMkdocsSetup.md:23]: 👍

File: docs/tremor-query/index.md [hunk 2]
```diff
@@ -124,9 +126,10 @@ with
 end;
 ```
 
-#### Customer Operator definitions
+#### Custom Operator definitions
 
-Custom operators allow legacy operators written before the query language was designed to be used with the query language. As the query langauge and legacy yaml format share the same DAG model and pipeline formats, they are interoperable at runtime and are backwards compatible:
+FIXME
```
Comment (mfelsche) [inline docs/tremor-query/index.md:131]: :scream:

File: docs/tremor-query/index.md [hunk 3]
```diff
@@ -185,25 +188,25 @@ select event from kfc into out;
...
 
-The select query is a builtin operation that is the workhorse of the tremor-query` language.
+The select query is a builtin operation that is the workhorse of the `tremor-query` language.
 
 The select operation is of the general form:
 
 > ![select grammar](grammar/diagram/SelectStmt.png) > ![from grammar](grammar/diagram/FromClause.png) > ![where grammar](grammar/diagram/WhereClause.png) > ![group by grammar](grammar/diagram/GroupByClause.png) > ![group by dimensions grammar](grammar/diagram/GroupByDimension.png) > ![set group grammar](grammar/diagram/SetBasedGroup.png) > ![each group grammar](grammar/diagram/EachBasedGroup.png) > ![into grammar](grammar/diagram/IntoClause.png) > ![having grammar](grammar/diagram/HavingClause.png)
 
-A example select operation configured to pass through data from a pipelines default `in` stream to a pipelines default `out` stream:
+A example select operation configured to pass through data from a pipeline's default `in` stream to a pipeline's default `out` stream:
```
Comment (mfelsche) [inline docs/tremor-query/index.md:197]:
```suggestion
An example select operation configured to pass through data from a pipeline's default `in` stream to a pipeline's default `out` stream:
```

File: docs/tremor-query/index.md [hunk 4]
```diff
@@ -124,9 +126,10 @@ with
 end;
 ```
 
-#### Customer Operator definitions
+#### Custom Operator definitions
 
-Custom operators allow legacy operators written before the query language was designed to be used with the query language. As the query langauge and legacy yaml format share the same DAG model and pipeline formats, they are interoperable at runtime and are backwards compatible:
+FIXME
+Custom operators allow legacy operators written before the query language. As the query langauge and deprecated yaml format share the same DAG model and pipeline formats, they are interoperable at runtime and are backwards compatible:
```
Comment (mfelsche) [inline docs/tremor-query/index.md:132]:
```suggestion
Custom operators allow definition, configuration and usage of legacy operators, that have been around before tremor supported the query language. As the query language and deprecated yaml format share the same DAG model and pipeline formats, they are interoperable at runtime and are backwards compatible:
```

**Review by mfelsche — COMMENTED** [review]
just 1 fixme still around, otherwise LGTM

File: docs/tremor-query/index.md (hunk 2 above)
Comment (anupdhml) [reply docs/tremor-query/index.md]: good catch! fixed it now.

**Review by mfelsche — APPROVED** [review]

**Pull request merged**